plugins/http_plugin/include/eosio/http_plugin/common.hpp

```diff
@@ -225,15 +158,17 @@ auto make_http_response_handler(std::shared_ptr<http_plugin_state> plugin_state,
          deadline = start + plugin_state->max_response_time;
       }
 
+      plugin_state->bytes_in_flight += payload_size;
+
       // post back to an HTTP thread to allow the response handler to be called from any thread
       boost::asio::post(plugin_state->thread_pool->get_executor(),
-                        [plugin_state, session_ptr, code, deadline, start,
-                         tracked_response = std::move(tracked_response)]() {
+                        [plugin_state, session_ptr, code, deadline, start, payload_size, response = std::move(response)]() {
                            try {
-                              if(tracked_response->obj().has_value()) {
-                                 std::string json = fc::json::to_string(*tracked_response->obj(), deadline + (fc::time_point::now() - start));
-                                 auto tracked_json = make_in_flight(std::move(json), plugin_state);
-                                 session_ptr->send_response(std::move(tracked_json->obj()), code);
+                              plugin_state->bytes_in_flight -= payload_size;
+                              if (response.has_value()) {
+                                 std::string json = fc::json::to_string(*response, deadline + (fc::time_point::now() - start));
+                                 if (session_ptr->verify_max_bytes_in_flight(json.size()))
```

Comment:
Remove this verify call.

Reply:
Why? It is needed to take into account the actual bytes in flight after converting to json.

Reply:
We discussed and going with the it as is.